.github/actions/spell-check/expect.txt

```diff
@@ -1642,42 +1292,27 @@ RESTORETOMAXIMIZED
 restrictedcapabilities
 restrictederrorinfo
 resultlist
-resw
-resx
-retval
-rfc
 RGBQUAD
 rgbs
 rgelt
 rgf
 rgn
 rgs
-rhs
-ricardosantos
 RIDEV
-RIGHTDOWN
 RIGHTSCROLLBAR
-RIGHTUP
 riid
 ringbuffer
 RKey
-RLO
-RMENU
 RNumber
 roadmap
 robmensching
-Roboto
-rooler
+robmikh
```

Comment:
This is a username. Why are we removing from names.txt and adding in expect.txt?